src/main/java/de/htwberlin/GradesOverview.java

```diff
@@ -1,27 +1,88 @@
 package de.htwberlin;
 
+import java.util.*;
+import java.util.stream.Collector;
+import java.util.stream.Collectors;
+import java.util.stream.Collectors.*;
+import java.util.stream.IntStream;
+import java.lang.Integer.*;
+
+import static java.lang.String.valueOf;
+import static java.util.stream.Collectors.counting;
+
 public class GradesOverview {
 
   private int[] grades;
+  private List<Integer> alternativeGrades = new ArrayList<>();
```

Comment:
In der Aufgabe ging es eigentlich darum, dass ausschliesslich mit dem Array gearbeitet wird.